sigstore/_cli.py

```diff
@@ -214,10 +201,10 @@ def _add_shared_verification_options(group: argparse._ArgumentGroup) -> None:
         required=True,
     )
     group.add_argument(
-        "--require-rekor-offline",
+        "--offline",
         action="store_true",
-        default=_boolify_env("SIGSTORE_REQUIRE_REKOR_OFFLINE"),
-        help="Require offline Rekor verification with a bundle; implied by --rekor-bundle",
+        default=_boolify_env("SIGSTORE_OFFLINE"),
+        help="Perform offline verification; requires a Sigstore bundle",
```

Comment:
I'm now realizing that this is slightly misleading, since `--offline` implies "fully" offline, whereas we still do some network requests for TUF. I think we should keep this flag, though, and work on removing those requests.